inference-engine/src/mkldnn_plugin/nodes/mkldnn_reduce_node.cpp

```diff
@@ -715,15 +975,15 @@ struct jit_uni_reduce_kernel_f32 : public jit_uni_reduce_kernel, public jit_gene
         }
     }
 
-    inline void load_embedded_horiz_reduce_store(Vmm vmm_dst, memory::data_type dst_dt) {
+    inline void horiz_reduce_store(Vmm vmm_dst, memory::data_type dst_dt, bool load_embedded = false) {
```

Comment:
could you please update to uni_xxx for this function.

Reply:
Fixed. Thanks Chenhu!